mslice/quickview/workspace_manager_quickview.py

```diff
@@ -17,6 +18,5 @@ def __getattribute__(self, item):
 
 
 if __name__ == '__main__':
-    from command import Command as c
```

Comment:
Is this file used anywhere? (Actually it seems the whole `quickview` module is kind-of orphaned... it seems it was used to mock up the ui when MSlice was first written 6 years ago and then not touched...)